tests/test_esdl_pycml.py

```diff
@@ -8,6 +8,14 @@
 
 class TestESDL(TestCase):
     def test_basic_source_and_demand_heat(self):
+        """
+        Test to check whether a hardcoded pycml model gives equivalent results compared to an esdl
+        specified model. Model used a source, pipe demand model.
```

Comment:
Maybe better wording: "The model consists out of a source, pipe and demand"?

Reply:
done